src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -22,7 +22,11 @@ import UIKit
 
 protocol CoronaWarnAppDelegate: AnyObject {
 	var client: HTTPClient { get }
-	var downloadedPackagesStore: DownloadedPackagesStore { get }
+	#if INTEROP
+	var downloadedPackagesStore: DownloadedPackagesStoreV1 { get }
+	#else
+	var downloadedPackagesStore: DownloadedPackagesStoreV0 { get }
+	#endif
```

Comment:
Couldn't we just define a typealias DownloadedPackagesStore with the correct class? Like:

```swift
#if INTEROP
typealias DownloadedPackagesStore = DownloadedPackagesStoreV1
#else
typealias DownloadedPackagesStore = DownloadedPackagesStoreV0
#endif
```

Defining this in one place reduces the amount of switches.

Reply:
🌟 Good Idea! This reduced the amount of INTEROP switches!